go/pkg/bertyaccount/service_account.go

```diff
@@ -393,10 +406,13 @@ func (s *service) getAccountMetaForName(accountID string) (*AccountMetadata, err
 	return meta, nil
 }
 
-func (s *service) DeleteAccount(_ context.Context, request *DeleteAccount_Request) (*DeleteAccount_Reply, error) {
+func (s *service) DeleteAccount(ctx context.Context, request *DeleteAccount_Request) (_ *DeleteAccount_Reply, err error) {
 	s.muService.Lock()
 	defer s.muService.Unlock()
 
+	_, _, endSection := tyber.Section(ctx, s.logger, fmt.Sprintf("Deleting account %s (AccountService)", request.AccountID))
+	defer func() { endSection(err, "") }()
```

Comment:
maybe you sometimes use the two first anonymous variables, but since I only see this usage, I suggest to add a dedicated helper for this most common way of using the app to increase the readability, something like:

```suggestion
	defer tyber.DeferableSection(ctx, s.logger, fmt.Sprintf("Deleting account %s (AccountService)", request.AccountID))(err, "")
```

or even

```suggestion
	defer tyber.DeferableSection(ctx, s.logger,
		fmt.Sprintf("Deleting account %s (AccountService)", request.AccountID),
	)(err, "")
```

Reply:
as you can see in the playground https://play.golang.org/p/ejXulLs39tt if you don't use a lambda, the error is bound at the start and so is nil

Reply:
I will add an helper without the two first returns